S7.Net/S7.Net.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   
   <PropertyGroup>
-    <TargetFrameworks>net452;net462;netstandard2.0;netstandard1.3;net5.0;net6.0;net7.0</TargetFrameworks>

```

Comment:
I don't know the policy for supporting older targets, but the same could apply to .NET 5 too.

My feeling is that for PLCs quite a lot of older targets are around, because they run and an running system isn't touched (in that area). 

So maybe NS 1.3 should be kept too (except everything from these API are covered by net452 or net462 -- which I don't know as NS 1.3 is soo old 😉).

Reply:
Sorry, this is not necessary. This happened when I was testing another code and the compilation failed. Later, I forgot to restore it.